sycl/.clang-format

```diff
@@ -1,2 +1,2 @@
 BasedOnStyle: LLVM
-NamespaceMacros: ['__SYCL_INLINE_NAMESPACE']
+NamespaceMacros: ['__SYCL_OPEN_NS','__SYCL_INLINE_NAMESPACE']
```

Comment:
Do we still need the old thing?

```suggestion
NamespaceMacros: ['__SYCL_OPEN_NS']
```

Reply:
Yes, so far it is still used for sycl builtins declaration.